### PR TITLE
Download pyo3-pack instead of installing.

### PR DESCRIPTION
**Pull request by sebpuetz**
I think that should be it

Addresses #65

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,19 +37,28 @@ install:
...
-  - cargo install --git https://github.com/PyO3/pyo3-pack.git --tag v0.6.1 pyo3-pack
   - rustup default nightly-2019-07-19
   - rustup component add rustfmt
   - rustup component add clippy
 
 script:
   - cargo fmt --all -- --check
   - cargo clippy -- -D warnings
-  - pyo3-pack develop
+  - |
```
Comment (danieldk) [inline .travis.yml:43]: Small comment: the if/download/unpack parts should probably be in the install section.

File: .travis.yml [hunk 2]
```diff
@@ -37,19 +37,28 @@ install:
...
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      wget https://github.com/PyO3/maturin/releases/download/v0.7.2/maturin-v0.7.2-x86_64-apple-darwin.tar.gz
```
Comment (danieldk) [inline .travis.yml]: We should probably start doing this properly. Prepend: `travis_retry`

File: .travis.yml [hunk 3]
```diff
@@ -37,19 +37,28 @@ install:
...
+      tar -xzf maturin-v0.7.2-x86_64-apple-darwin.tar.gz
+    fi
+  - |
+    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+      wget https://github.com/PyO3/maturin/releases/download/v0.7.2/maturin-v0.7.2-x86_64-unknown-linux-musl.tar.gz
```
Comment (danieldk) [inline .travis.yml]: Same here.

**Review by danieldk — CHANGES_REQUESTED** [review]

**Comment by sebpuetz** [comment]
Done. Looks like build time is down from ~35 mins to ~10 mins by downloading the binary!

**Review by danieldk — APPROVED** [review]

**Pull request merged**